libs/shared/node/src/circle-ci-api/circle-ci-api.ts

```diff
@@ -128,6 +131,32 @@ export class CircleCiApi extends ContinuosIntegrationApi {
       );
   }
 
+  getCommitShaOfLastSuccessfulRun(branchName: string): Observable<string> {
+    return this._getLastSuccessfulRun(branchName).pipe(
+      pluck('id'),
```

Comment:
NIT: We could omit pluck and use a simple map and optional chaining :) https://twitter.com/BenLesh/status/1245876048128073728